# Control mode with limiting parameters

uint8 mode	#0 - Ackermann, 1 - turn-in-place, 2 - crab
float32 linear_speed_limit
float32 smth_else
float32 angle_accuracy
float32 distance_accuracy
